### ISSUE TITLE
integration with Gutenberg map blocks

### ISSUE THREAD

**Issue by jywarren** [publiclab/leaflet-blurred-location#162]
https://jetpack.com/support/jetpack-blocks/map-block/

**Comment by sagarpreet-chadha** [comment]
Hi @jywarren , i think this is only for WordPress websites .

**Comment by jywarren** [comment]
Yes and we can ignore this for now... I've been thinking about long term
using Gutenberg as part of publiclab.editor... but it's a long way off!
